featurebyte/service/feature_list.py

```diff
@@ -320,6 +328,12 @@ async def _get_feature_list_version(self, name: str) -> VersionIdentifier:
         count = query_result["total"]
         return VersionIdentifier(name=version_name, suffix=count or None)
 
+    @retry(
```

Comment:
Is this retry still needed with the optimised query?

Reply:
Since the transaction here involves a number of queries, just add a retry here to avoid having issue when the mongo is busy.